blues/ncmc.py

```diff
@@ -534,44 +533,23 @@ def runSim(self, md_simulation, nc_context, nc_integrator, dummy_simulation, mov
                 except Exception as e:
                     print('Error:', e)
                     stateinfo = md_simulation.context.getState(True, True, False, False, False, periodic)
-                    print(oldPos)
                     last_x, last_y = np.shape(oldPos)
-                    print(np.reshape(oldPos, (1, last_x, last_y)))
                     reshape = (np.reshape(oldPos, (1, last_x, last_y))).value_in_unit(unit.nanometers)
-                    print(np.shape(reshape))
-                    print(newinfo.getPotentialEnergy())
-                    print(newinfo.getKineticEnergy())
                     print(oldPE)
```

Comment:
Are these prints debug statements? Usually not very useful to user to have a print of something without hvaing it say what is being printed.